frontend/device/android/device.lua

```diff
@@ -375,6 +377,31 @@ function Device:canExecuteScript(file)
     end
 end
 
+function Device:showLightDialog()
+    local usleep = require("ffi/util").usleep
+    local title = android.isEink() and _("Frontlight settings") or _("Light settings")
+    android.lights.showDialog(title, _("Brightness"), _("Warmth"), _("Ok"), _("Cancel"))
```

Comment:
```suggestion
    android.lights.showDialog(title, _("Brightness"), _("Warmth"), _("OK"), _("Cancel"))
```

To be clear, this shows up as the intended
Cancel/OK
?

Reply:
OK